data/json/npcs/isolated_road/isolated_road_jay_dialogue.json

```diff
@@ -195,14 +195,15 @@
     "id": "TALK_GUNSMITH_KORD",
     "type": "talk_topic",
     "dynamic_line": [
-      "Where did you… I- I don't know what to say.\"  Jay takes the rifle reverently.  He turns it over so carefully that you could mistake him for holding a child instead of a gun.  \"This thing is is beautiful.  Modern rails, ergonomic furniture, recoil balancing system.  I know Cody thinks that she did something wrong, but all I remember is being so angry… just blindingly furious.  Then I was running.  When I woke up, everything was fine.\"\n\nJay hands back the Kord.  \"Thank you for letting me see this, and for getting Cody and I talking.  If you could help me out with something for Cody, I'll hook you up with a casket mag for that thing and a box of the good ammo.  Stuff from my reserves."
+      "Where did you… I- I don't know what to say.\"  Jay takes the rifle reverently.  \"What did they do this thing?  The receiver's all chopped up!\"  He turns it over so carefully that you could mistake him for holding a child instead of a gun.  \"This thing was beautiful.  Modern rails, ergonomic furniture, recoil balancing system… At least it's all still here.  I know Cody thinks that she did something wrong, but all I remember is being so angry… just blindingly furious.  Then I was running.  When I woke up, everything was fine.\"\n\n\"Thank you for letting me see this, and for getting Cody and I talking.  If you could help me out with something for Cody, I'll make this rifle right, and hook you up with a casket mag and a box of the good ammo for it.  Stuff from my reserves."
```

Comment:
```suggestion
      "Where did you… I- I don't know what to say.\"  Jay takes the rifle reverently.  \"What did they do to this thing?  The receiver's all chopped up!\"  He turns it over so carefully that you could mistake him for holding a child instead of a gun.  \"This thing was beautiful.  Modern rails, ergonomic furniture, recoil balancing system… At least it's all still here.  I know Cody thinks that she did something wrong, but all I remember is being so angry… just blindingly furious.  Then I was running.  When I woke up, everything was fine.\"\n\n\"Thank you for letting me see this, and for getting Cody and I talking.  If you could help me out with something for Cody, I'll make this rifle right, and hook you up with a casket mag and a box of the good ammo for it.  Stuff from my reserves."
```